<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devices_menu</class>
 <widget class="QWidget" name="devices_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="device_menu" native="true">
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="save_config_button">
        <property name="text">
         <string>Save Config</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="load_config_button">
        <property name="text">
         <string>Load Config</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
